frontend/index.html

```diff
@@ -39,15 +39,48 @@
                     <span>Ideas & Advice</span></div>
             </a>
             
-            <a class="nav__icons" href="#basket">
+            <button class="nav__icons" id="cart-icon">
                 <div class="nav__icons__item container"> <img img class="searchicons" src="./assets/images/basket.svg" alt="basket">
                     <span>Basket</span>
                 </div>  
-            </a>
+            </button>
             <div class="nav__search-container">
                 <input class="search-bar" type="search" placeholder="Search...">
             </div> 
-        </div>                            
+        </div>
+        
+        <div class="cart">
+            <h2 class="card-title">Your Cart</h2>
+
+            <div class="cart-content">
+                <template id="cart-box-template">
+                    <div class="cart-box">
+                        <img src="./assets/images/trending-product.webp" alt=" ", class="cart-img">
+                        <div class="cart-box__detail">
+                            <div class="cart-box__product-detail">Item Name</div>
+                            <div class="cart-box__product_price">£25.05</div>
+                            <div class="cart-box_quantity-control">
+                                <button class="cart-box_quantity-decrease"><img class="cart-box-btn__disabled" src="./assets/images/dash-square.svg" alt=""></button>
```

Comment:
I noticed a small difference in disabling the increase and decrease buttons respectively, where the class is applied to the image in this case, and to the button itself in the other

Increase:
![IncreaseDisabled](https://github.com/MinhazH99/orders101/assets/1670145/c7b047dd-2adf-4c91-970c-ceb2be6fc5ba)

Decrease:
![DecreaseDisabled](https://github.com/MinhazH99/orders101/assets/1670145/9757d8d4-ecbf-4ee3-ad49-286334962c9b)

 If we were to add the `aria-disabled` attribute [as mentioned here](https://github.com/MinhazH99/orders101/pull/59/files#r1538992312), would it make sense to do that for the buttons in both cases?